compiler/passes/errorHandling.cpp

```diff
@@ -17,8 +17,185 @@
  * limitations under the License.
  */
 
+#include "AstVisitorTraverse.h"
```

Comment:
#include "foo.h" should always be the upper most #include in foo.cpp (to avoid a situation in which
foo.h has a hidden dependency on some other .h file